build.sbt

```diff
@@ -24,6 +24,7 @@ inThisBuild(
 )
 
 addCommandAlias("build", "; prepare; testJVM")
+addCommandAlias("prepare", "fix")
```

Comment:
Why do we need this alias?

Reply:
Immediate need is to fix the `build` alias as it's broken at the moment. Personally, I wouldn't mind fixing it by just using `fmt` directly, but @sideeffffect suggested that we should keep this for extensibility.

P.S. Now that I'm responding to you I realize there's an error in it (used `fix` instead of `fmt`).

Reply:
I would suggest we just use `fmt` for everything.